feature/widget/shortcut/src/main/kotlin/app/k9mail/feature/widget/shortcut/LauncherShortcutActivity.kt

```diff
@@ -9,15 +9,20 @@ import androidx.annotation.AttrRes
 import androidx.core.content.pm.ShortcutInfoCompat
 import androidx.core.content.pm.ShortcutManagerCompat
 import androidx.core.graphics.drawable.IconCompat
+import app.k9mail.core.common.provider.AppNameProvider
 import com.fsck.k9.BaseAccount
+import com.fsck.k9.activity.AccountList
+import com.fsck.k9.activity.MessageList
 import com.fsck.k9.search.SearchAccount
-import com.fsck.k9.ui.R
+import org.koin.android.ext.android.inject
 import app.k9mail.core.ui.legacy.theme2.common.R as CommonR
 
-class LauncherShortcuts : AccountList() {
+class LauncherShortcutActivity : AccountList() {
+    private val appNameProvider: AppNameProvider by inject()
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setTitle(R.string.shortcuts_title)
+        setTitle(resources.getString(R.string.shortcut_widget_title, appNameProvider.appName))
```

Comment:
Having the app name here feels unnecessary. I think it's fine to just use "Accounts" for the activity title.

Reply:
At the moment all widgets use the app name for their configuration interface. We could of course entirely remove the app name.

Reply:
As the translations are using a form of `K-9 Accounts` I need to drop the translations then, as they are not correct anymore.

Reply:
:+1: